src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -467,13 +467,25 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
           source.setAttribute('type', mimetype);
 
           if (this._config.useMediaOptionAttribute) {
-            const options = {};
-            options.option = {};
+            let options = {};
+            // https://webostv.developer.lge.com/develop/guides/mediaoption-parameter
+            if (this._config.mediaOptionAttribute) {
+              /**
```

Comment:
remove obsolete code

Reply:
this is also documentation for possible usage of the config, I think this is good to keep as it is undocumented